edx_exams/apps/api/serializers.py

```diff
@@ -97,6 +97,28 @@ class Meta:
         )
 
 
+class AssessmentControlReviewSerializer(serializers.ModelSerializer):
+    """
+    Serializer for AssessmentControlResult as a review. Currently a single
+    ACS response is equvalent to a review but this may change if we introduce
+    multiple ACS responses per attempt.
```

Comment:
Just double checking, did we decide with Proctorio that there will only be one ACS response per attempt (as denoted in the serializer below)? Would there be any reason we'd want to maybe futureproof things by allowing there to be more than one ACS response/review per exam attempt?

Reply:
kinda like my other comment, I wanted to future proof the database design because that's hard to change but the UI presentation I just left as simple as possible since that's easy to rework to fit whatever the requirements are for multiple types of ACS responses.